src/main/java/cz/cvut/fel/aic/simod/ridesharing/insertionheuristic/DriverPlan.java

```diff
@@ -29,12 +34,16 @@
  * @author F.I.D.O.
  */
 public class DriverPlan implements Iterable<PlanAction>{
```

Comment:
This class should not be used for plan creation, it is an output class for the simulation. Create your own data structures for this instead.